cloudformation/stacks/AutoSpotting/parameters.yaml

```diff
@@ -1,3 +1,8 @@
 ---
 LambdaHandlerFunction: handler.handle
 LambdaS3Bucket: cloudprowess
+LambdaMemorySize: '256'
+ExecutionFrequency: "rate(1 minute)"
+MinOnDemandNumber: '1'
+MinOnDemandPercentage: '50'
+Regions: us-east-1,eu-west-1,ca-central-1
```

Comment:
We currently have no checks for regions, while it would be used at runtime. It might be better to whether make a call to `describe-regions` to have something dynamic, or use `const` with a list of current region - but require to keep it updated. I'd be more in favour of the API call, as it wouldn't require commits whenever a change of region/region name occurs.

As we are introducing it to runtime, it might also be the opportunity to make it more flexible via describe-regions, and use globing as they do: `aws ec2 describe-regions --filters "Name=endpoint,Values=*us*"`
Src: http://docs.aws.amazon.com/sdk-for-go/api/service/ec2/ & http://docs.aws.amazon.com/cli/latest/reference/ec2/describe-regions.html. This way we could easily set the lambda to run a different regions.